### ISSUE TITLE
System Trust

### ISSUE THREAD

**Issue by Chayotea** [pypy-vrc/VRCX#337]
Seems system_legend and system_trust_legend are gone now. Users that were Veteran now will be trusted when clicked on. They return to Veteran when VRCX is restarted. Seems VRCX is not saving this change.

**Comment by Myrkie** [comment]
its being worked on.

**Comment by SalbugVR** [comment]
Natsumi gotta update VRCX to reflect the recent API changes. :v

**Comment by GenesisAria** [comment]
It also spams the friend log which is the real problem.

**Comment by SonGorLick** [comment]
Big gay they removed veteran

**Comment by Myrkie** [comment]
Veteran and Legendary were dropped ontop of some others. doesn't matter they were legacy anyway.

**Comment by SonGorLick** [comment]
Still it showed up for people who had this or ogtrustrank and there was no reason to remove it

**Comment by Myrkie** [comment]
its legacy there is no point in keeping it, it did nothing for the game or the API only way to see it is using third party software or mods which are against terms of service.

**Comment by SalbugVR** [comment]
I mean, no reason to keep around something that was deprecated and wasn't being used officially. ¯\_(ツ)_/¯

**Comment by SonGorLick** [comment]
Again there was no reason to remove it either

**Comment by Myrkie** [comment]
still, no reason to keep it no reason to remove it. they decided to remove it and that's where we are now.

**Comment by SalbugVR** [comment]
I'm pretty sure the reason was to tidy up things on the back-end. No reason to keep deprecated things around to add to the clutter unless you're a hoarder I guess. 🤔

**Comment by SonGorLick** [comment]
Dosent really clutter anything up, if they where so worried about that, theres so many other things they could do

**Comment by Myrkie** [comment]
what else could they do if you want to elaborate?

**Comment by SonGorLick** [comment]
What dose them removing a rank do that improves the game, if they cared about ogtrust ranks braking tos or other mods that improved on the game, maybe they should fix all the problems with the game first, as well as emm brakes tos but vrc dosent mind anymore because you have to buy vrc+ to use the fav's. 

For the rank thing, they could have just brought it back or added new ranks or something idk i see no reason on why they would remove it after so long of it sitting there and a massive amount of people knowing about the rank.

**Comment by information-redacted** [comment]
> What dose them removing a rank do that improves the game, if they cared about ogtrust ranks braking tos or other mods that improved on the game, maybe they should fix all the problems with the game first, as well as emm brakes tos but vrc dosent mind anymore because you have to buy vrc+ to use the fav's.
> 
> For the rank thing, they could have just brought it back or added new ranks or something idk i see no reason on why they would remove it after so long of it sitting there and a massive amount of people knowing about the rank.

Cringe. Just let them do their damn thing, and stop caring about a deprecated tag. If it means that much to you, you should think about your priorities and your apparent god complex :)

**Comment by Myrkie** [comment]
the ranks were never officially in the game they were for the small best period during the rank beta but never actually made it into the game that's around 4 years of it having no point of existing, and no just because emm requires you to buy VRC+ to use extended favorites != to being approved by vrchat

**Comment by SonGorLick** [comment]
> > What dose them removing a rank do that improves the game, if they cared about ogtrust ranks braking tos or other mods that improved on the game, maybe they should fix all the problems with the game first, as well as emm brakes tos but vrc dosent mind anymore because you have to buy vrc+ to use the fav's.
> > For the rank thing, they could have just brought it back or added new ranks or something idk i see no reason on why they would remove it after so long of it sitting there and a massive amount of people knowing about the rank.
> 
> Cringe. Just let them do their damn thing, and stop caring about a deprecated tag. If it means that much to you, you should think about your priorities and your apparent god complex :)

Big cringe removing something they should have removed forever ago and just did it now

**Comment by Myrkie** [comment]
and? they have only recently gotten more competent aswell as a extended number of staff for the API and Client development team, obviously it wasn't a priority then look what they have done between then and now to make the API less viable to scraping / avatar asseturl yoinking.

**Comment by SonGorLick** [comment]
> and? they have only recently gotten more competent aswell as a extended number of staff for the API and Client development team, obviously it wasn't a priority then look what they have done between then and now to make the API less viable to scraping / avatar asseturl yoinking.

What dose the Veteran rank have anything to do with yoinking

**Comment by Myrkie** [comment]
> > and? they have only recently gotten more competent aswell as a extended number of staff for the API and Client development team, obviously it wasn't a priority then look what they have done between then and now to make the API less viable to scraping / avatar asseturl yoinking.
> 
> What dose the Veteran rank have anything to do with yoinking
 
 your trying to invalidate there there work because they decided to do some spring cleaning and your complaining about it, either way the tag removal is a Non issue, after all its an undocumented API that is subject to change at anytime without notice.

**Comment by SonGorLick** [comment]
> > > and? they have only recently gotten more competent aswell as a extended number of staff for the API and Client development team, obviously it wasn't a priority then look what they have done between then and now to make the API less viable to scraping / avatar asseturl yoinking.
> > 
> > 
> > What dose the Veteran rank have anything to do with yoinking
> 
> your trying to invalidate there there work because they decided to do some spring cleaning and your complaining about it, either way the tag removal is a Non issue, after all its an undocumented API that is subject to change at anytime without notice.

You just said it had something to do with yoinking, now your moving on from it, two theres years late for the so called spring cleaning of removing something that should have been gone forever ago

**Comment by Myrkie** [comment]
> > > > and? they have only recently gotten more competent aswell as a extended number of staff for the API and Client development team, obviously it wasn't a priority then look what they have done between then and now to make the API less viable to scraping / avatar asseturl yoinking.
> > > 
> > > 
> > > What dose the Veteran rank have anything to do with yoinking
> > 
> > 
> > your trying to invalidate there there work because they decided to do some spring cleaning and your complaining about it, either way the tag removal is a Non issue, after all its an undocumented API that is subject to change at anytime without notice.
> 
> You just said it had something to do with yoinking, now your moving on from it, two theres years late for the so called spring cleaning of removing something that should have been gone forever ago

let me rephrase then, VRChat has actively been busy with other parts of the API, securing the API against third party Yoinking of asset bundles, no longer publicly exposing asset URLs to non client requests. now they have got to spring cleaning and your saying they can do better because they decided to remove some old tags, they are doing their job why are you whining about it? once again a undocumented API that is not even supposed to be looked at by people they just don't care if its being interacted with.

![image](https://user-images.githubusercontent.com/20288698/166860526-1e86c997-4580-45d9-a111-5fafbadf7e64.png)

**Comment by SonGorLick** [comment]
> > > > > and? they have only recently gotten more competent aswell as a extended number of staff for the API and Client development team, obviously it wasn't a priority then look what they have done between then and now to make the API less viable to scraping / avatar asseturl yoinking.
> > > > 
> > > > 
> > > > What dose the Veteran rank have anything to do with yoinking
> > > 
> > > 
> > > your trying to invalidate there there work because they decided to do some spring cleaning and your complaining about it, either way the tag removal is a Non issue, after all its an undocumented API that is subject to change at anytime without notice.
> > 
> > 
> > You just said it had something to do with yoinking, now your moving on from it, two theres years late for the so called spring cleaning of removing something that should have been gone forever ago
> 
> let me rephrase then, VRChat has actively been busy with other parts of the API, securing the API against third party Yoinking of asset bundles, no longer publicly exposing asset URLs to non client requests. now they have got to spring cleaning and your saying they can do better because they decided to remove some old tags, they are doing their job why are you whining about it? once again a undocumented API that is not even supposed to be looked at by people they just don't care if its being interacted with.
> 
> ![image](https://user-images.githubusercontent.com/20288698/166860526-1e86c997-4580-45d9-a111-5fafbadf7e64.png)

Again the ranking shit has nothing to do with yoinking and should have been removed long ago, i bet they had to back space a few lines for the ranks. None of what you just sent explains how the ranking has anything to do with yoinking, every update they push they fuck up the game more then it was before the update. yet again im asking you wtf the old ass ranks have to do with yoinking or anything it dosent, the moment these old ranks wernt making it into the game they should have removed it

**Comment by information-redacted** [comment]
Hi,

Do everyone a favor and take this to the [Discord](https://discord.gg/DBzR9sZa), not everyone subscribed to this thread should have this bickering.

**Comment by Myrkie** [comment]
> Hi,
> 
> Do everyone a favor and take this to the [Discord](https://discord.gg/DBzR9sZa), not everyone subscribed to this thread should have this bickering.

^

**Comment by DubyaDude** [comment]
lul, good riddance

**Comment by Natsumi-sama** [comment]
Ranks/Tags gone,
